### ISSUE TITLE
Store screener questions for users who don't enter Paypal info

### ISSUE THREAD

**Issue by DukeCityDigital** [DukeCityDigital/mysurveys#195]
5) If some participants complete the screening questions but do not provide their paypal data, could we still store their screener answers, and e.g. have the paypal status recorded as ‘None’?

**Comment by DukeCityDigital** [comment]
@olssonresearch Yeah it looks like you can see the screener answers in the data export even if the user didn't verify Paypal - lmk if this is what is meant

**Comment by olssonresearch** [comment]
@DukeCityDigital Yes this works! It is useful for us to have seeds' answers to the screener even if they didn't verified PayPal
